core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -514,18 +514,11 @@ class RealityTreeReference extends RealityModelTileTree.Reference {
     if (undefined === tree)
       return undefined;
 
-    if (!this._iModel.isGeoLocated || !tree.isContentUnbounded)
-      return super.createDrawArgs(context);
-
-    const now = BeTimePoint.now();
-    const location = this.computeTransform(tree);
-    const clipVolume = this.getClipVolume(tree);
-    const viewFlagOverrides = this.getViewFlagOverrides(tree);
-
-    location.origin.z += context.viewport.displayStyle.backgroundMapElevationBias;
-
-    return new TileDrawArgs({ context, location, tree, now, viewFlagOverrides, clipVolume, parentsAndChildrenExclusive: tree.parentsAndChildrenExclusive, symbologyOverrides: this.getSymbologyOverrides(tree) });
+    let drawArgs = super.createDrawArgs(context);

```

Comment:
Linter's going to complain about `let` instead of `const`.

Reply:
Fixed.